go/cmd/topo2topo/topo2topo.go

```diff
@@ -18,35 +18,29 @@ package main
 
 import (
 	"context"
-	"flag"
 	"fmt"
 	"os"
 
-	"vitess.io/vitess/go/vt/servenv"
-
 	"github.com/spf13/pflag"
 
 	"vitess.io/vitess/go/exit"
 	"vitess.io/vitess/go/vt/grpccommon"
 	"vitess.io/vitess/go/vt/log"
 	"vitess.io/vitess/go/vt/logutil"
+	"vitess.io/vitess/go/vt/servenv"
 	"vitess.io/vitess/go/vt/topo"
 	"vitess.io/vitess/go/vt/topo/helpers"
-
-	// Include deprecation warnings for soon-to-be-unsupported flag invocations.
-	_flag "vitess.io/vitess/go/internal/flag"
 )
 
 func main() {
 	defer exit.RecoverAll()
 	defer logutil.Flush()
 
 	fs := pflag.NewFlagSet("topo2topo", pflag.ExitOnError)
```

Comment:
in order make the flags registered to this flag set show up, you'll need to:

```go
servenv.OnParse(func(flags *pflag.FlagSet) { flags.AddFlagSet(fs) })
```

in order to move the flags over into the one that servenv is going to actually use.

alternatively, you can inline the registration, like:

```go
servenv.OnParse(func(fs *pflag.FlagSet) {
    fs.StringVar(&fromImplementation, "from_implementation", "blah blah blah", ...)
    ... 
})
```

Reply:
Fixed